src/main.rs

```diff
@@ -173,11 +181,12 @@ fn main() {
 }
 
 /// Configures the Actix application
-fn configure_app(app: App<MiniserveConfig>) -> App<MiniserveConfig> {
+fn configure_app(mut app: App<MiniserveConfig>) -> App<MiniserveConfig> {
```

Comment:
Do you really need this mut?

Reply:
Probably not, I changed it and I think it is better now.